python/ray/serve/config.py

```diff
@@ -263,84 +265,27 @@ def __init__(
         if ray_actor_options is None:
             self.ray_actor_options = {}
         else:
+            if not isinstance(ray_actor_options, dict):
+                raise TypeError("ray_actor_options must be a dictionary.")
+            allowed_ray_actor_options = {
```

Comment:
This means if new options are added to Ray tasks and actors, Serve automatically defaults to disallowing them. Is that expected? cc @edoakes

Reply:
Yes this sounds good, we should explicitly whitelist instead of blacklist